priv/catalog/9FAAD0.yaml

```diff
@@ -0,0 +1,23 @@
+id: "9FAAD0"
+name: unsupported pacemaker version
+group: OS and package versions
+description: |
+  Pacemaker version is not the recommended value
+remediation: |
+  ## Abstract
+  Installed Pacemaker version must not be equal than the recommended value
+
+  ## Remediation
+  Install or upgrade to a supported Pacemaker version
+
+  ## Reference
+  - https://documentation.suse.com/en-us/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/
+
+facts:
+  - name: exclude_package_pacemaker
+    gatherer: package_version
+    argument: pacemaker,2.0.3+20200511.2b248d828
```

Comment:
This could very well done as before, just letting the version in the values, and comparing with the package version string, as we simply wanted a different value.
But both work